ui2/src/icons/icons.tsx

```diff
@@ -17,6 +17,18 @@ export const ArrowRightIcon = () => (
     </svg>
 )
 
+export const CogIcon = () => (
+    <svg width="19" height="20" viewBox="0 0 19 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
just for the sake of following the pattern here - `className="icon cog-icon"` and you probably only need the viewbox